tldw_Server_API/app/core/Evaluations/recipe_runs_jobs_worker.py

```
@@ -0,0 +1,1036 @@
+    if not db_path:
+        db_path = str(DatabasePaths.get_evaluations_db_path(user_id))
+    return EvaluationsDatabase(db_path)
+
+
+def _get_service(*, user_id: str | None, db: EvaluationsDatabase | None = None) -> RecipeRunsService:
+    if db is not None:
+        return RecipeRunsService(db=db, user_id=user_id)
+    return get_recipe_runs_service_for_user(user_id)
+
+
+def _build_media_db(user_id: str) -> Any:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_build_media_db`

(ANN401)